css/styles.css

```diff
@@ -0,0 +1,78 @@
+/*
+- Font sizes (px):
+10 / 12 / 14 / 16 / 18 / 20 / 24 / 30 / 36 / 44 / 52 / 62 / 74 / 86 / 98
+
+- Spacing system (px)
+2 / 4 / 8 / 12 / 16 / 24 / 32 / 48 / 64 / 80 / 96 / 128
+
+*/
+
+:root {
+  font-size: 10px;
+  font: 400 10px / 1.2 "Yeseva One";
+}
+
+.container {
+  max-width: 160rem;
+  margin: 0 auto;
+}
```

Comment:
You're using `container` class inside of `header`. 
`container` is too common name, it's better to name it `header-container` in this case. But actually the properties look like it's a global container that's intended to wrap the whole body inner content.